src/com/oakesville/mythling/prefs/PrefsActivity.java

```diff
@@ -80,6 +80,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    @Override
+    protected boolean isValidFragment(String fragmentName) {
```

Comment:
got a runtime exception after upgrading to sdk 21. I am just returning true here to satisfy an 'unimplemented method' error. When debugging the fragmentName parameter, the value was "com.oakesville.mythling.prefs.NetworkPrefs".

I don't have anything valuable to 'validate' here basically so just return 'true'